tests/integration/solidity_contracts/PlainOpcodes/PlainOpcodes.sol

```diff
@@ -127,4 +128,22 @@ contract PlainOpcodes {
             extcodecopy(target, add(extcode, 0x20), offset, size)
         }
     }
+
+    function testCallingContextShouldPropogateRevertFromSubContextOnCreate() public {
+        ContractRevertsOnConstruction doomedContract = new ContractRevertsOnConstruction();
+        doomedContract.value();
```

Comment:
if it's reverting on construction, this line is never reached right?

Reply:
yeah, but necessary to get rid of a linting error of an used variable

Reply:
can't you just write

```
function testCallingContextShouldPropogateRevertFromSubContextOnCreate() public {
    new ContractRevertsOnConstruction();
}
```